examples/firefly.rs

```diff
@@ -70,6 +67,14 @@ impl FireflyWorld {
         self.firefly_swarm.push(firefly);
     }
 
+    fn getDist(fireFlyA:&Firefly, fireFlyB: &Firefly) -> f32 {
```

Comment:
The naming here should be changed to better conform to rust's style. Try changing this line to 
```rust
fn get_dist(firefly_a: &Firefly, firefly_b: &Firefly) -> f32
```

Reply:
Actually, a more ergonomic way would be to make this a function of firefly, but that's a matter of opinion. Something like
```rust
impl Firefly {
    fn distance_to(&self, &other: Firefly) -> f32 {
        ...
    }
}
```
But I'm not 100% sure. @sremedios  opinions?

Reply:
I will fix naming to better conform to rust style :rust: ! 
regarding making it a function of firefly - @sremedios ?